.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
- [ ] It would be great to highlight a piece of code you are proud of making or you find interesting in the video presentation

Reply:
- [ ] Requirement met